android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/RegisterFragment.kt

```diff
@@ -184,6 +184,12 @@ class RegisterFragment : Fragment(), OnSyncListener, Observer<QuestionnaireSubmi
             SnackBarMessageConfig(message = getString(R.string.syncing))
           )
         }
+      is SyncJobStatus.InProgress ->
+        lifecycleScope.launch {
```

Comment:
is it possible to write a test that gets at this? Lifecycle stuff seems hard to test

Reply:
A little refactoring has got us the desired result :)